File: baekjoon/p04796/Main.java
package baekjoon.p04796;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int i = 1;

        while (true) {
            int l = sc.nextInt();
            int p = sc.nextInt();
            int v = sc.nextInt();
            if (l == 0 && p == 0 && v == 0) {
                break;
            }

            int solve = solve(l, p, v);
            System.out.println(String.format("Case %d: %d", i++, solve));
        }
    }

    private static int solve(int l, int p, int v) {
        int ret = 0;

        ret += (v / p) * l;
        ret += Math.min(v % p, l);

        return ret;
    }

}

File: baekjoon/p04796/Main.kt
package baekjoon.p04796

fun main() {
    var i = 1
    while (true) {
        val inputs = readLine()!!.split(" ").map { it.toInt() }
        if (inputs.all { it == 0 }) {
            break
        }

        val solve = solve(inputs[0], inputs[1], inputs[2])
        println("Case ${i++}: $solve")
    }
}

private fun solve(l: Int, p: Int, v: Int): Int {
    var ret = 0

    ret += (v / p) * l
    ret += minOf(v % p, l)

    return ret
}